metadata-models/src/main/pegasus/com/linkedin/ingestion/DataHubIngestionSourceInfo.pdl

```diff
@@ -56,5 +56,10 @@ record DataHubIngestionSourceInfo {
       * Whether or not to run this ingestion source in debug mode
       */
       debugMode: optional boolean
+
+     /**
+      * Extra arguments for the ingestion run.
+      */
+      extraArgs: optional map[string, string]
```

Comment:
did you have a specific reason you wanted this to be a loose map instead of several stronger typed fields for the new things you want to add?

Reply:
It maps directly to the underlying PDL model, I didn't want to be prescriptive here. Do you have a strong opinion on this?

Reply:
nope that's cool with me